cfgov/v1/atomic_elements/atoms.py

```diff
@@ -154,3 +155,33 @@ def clean(self, data):
             raise ValidationError("ImageBasicUrlAlt validation errors", params=error_dict)
         else:
             return data
+
+class AnchorLink(blocks.StructBlock):
+    link_id = blocks.CharBlock(required=False)
```

Comment:
Separately, I would also move this block definition outside of `atoms` since it's not part of the atomic design and therefore shouldn't be defined as such. You could put it in `v1/blocks.py`.